components/server/src/ome/services/sharing/ShareBean.java

```diff
@@ -752,6 +758,35 @@ public void invalidateConnection(long shareId, Experimenter exp) {
         throw new UnsupportedOperationException();
     }
 
+    @RolesAllowed("user")
+    @Transactional(readOnly = false)
+    public void notifyMembersOfShare(long shareId, String subject, String message,
+            boolean html, List<Experimenter> exps) {
+
+        String sender = mailUtil.getSender();
+        if (StringUtils.isBlank(sender)) {
+            log.error("omero.mail.from cannot be empty.");
```

Comment:
I see what you mean by this, but I'm generally worried that if some script gets a hold of this the sysadmin could get numerous ERROR reports that really come down to a single configuration issue. Could `mailUtil` log this once at startup?